ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/AddMetadata.java

```diff
@@ -42,7 +42,8 @@
 
   private static final String HEADER = "header";
   private static final List<String> HEADER_ATTRIBUTES = ImmutableList //
-      .of(Attribute.DATE, Attribute.DNT, Attribute.X_PINGSENDER_VERSION, Attribute.X_DEBUG_ID);
+      .of(Attribute.DATE, Attribute.DNT, Attribute.X_PINGSENDER_VERSION, Attribute.X_DEBUG_ID,
+          Attribute.X_SOURCE_TAGS);
```

Comment:
I'm assuming here that we'll include x_debug_id as a simple string field in BigQuery, and potentially update user-facing views to transform the string to a list. This seems cleaner than trying to parse the field to a list within the pipeline.